index.bs

```diff
@@ -326,6 +269,24 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
             {{MediaEncodingType}} to which it's not applicable, return
             <code>false</code> and abort these steps. See applicability rules
             in the member definitions below.
+          </li>
+          <li>
+            Let |mimeType| be the result of running [=parse a MIME type=]
+            with |configuration|'s {{VideoConfiguration/contentType}}.
+          </li>
+          <li>
+            If |mimeType| is <code>failure</code>, return <code>false</code> and
+            abort these steps.
```

Comment:
```suggestion
            If |mimeType| is `failure`, return `false` and
            abort these steps.
```

Reply:
This spec seems to have a mix of markup styles.  I am happy to fix for this PR but we need to do a spec-wide cleanup on this.